tests/unit/test_row.py

```diff
@@ -833,3 +855,5 @@ def __init__(self, name, client=None, app_profile_id=None):
 
     def mutate_rows(self, rows):
         self.mutated_rows.extend(rows)
+        from google.rpc import status_pb2
+        return [status_pb2.Status()]
```

Comment:
Please move the import to the top of the method, and return an actual status code, e.g.:

```python
    def mutate_rows(self, rows):
        from google.rpc import status_pb2
        
        self.mutated_rows.extend(rows)
        return [status_pb2.Status(code=0)]
```

Reply:
Done.